src/components/charts/DataTable.vue

```diff
@@ -56,25 +65,75 @@ export default {
     async updateData () {
       const data = await util.getConfigCijfers(this.gwb, this.config)
 
+      // temp var for demo purposes; will be removed after client approval
```

Comment:
Does this project use vuex? This bit of code here would typically reside in a vuex store getter. The basic idea is this:

1. Component triggers the vuex action that fetches server data
2. The vuex store retrieves the data and saves it to state
3. Component maps a store getter containing all the logic enclosed in this method

A vuex store getter is a lens for viewing a state property. It is the ideal place to enrich / change server fetched data.

Reply:
The application is indeed using Vuex. I agree with your suggestion, but it's out of scope of the current assignment. I'll make a ticket for it, so that it can be addressed at a later date.